wizards/Policy/PolicyWizard.tsx

```diff
@@ -338,6 +339,31 @@ export function PolicyWizardTemplates(props: { policies: IResource[] }) {
                         validation={isValidKubernetesResourceName}
                     />
 
+                    <WizRadioGroup
+                        path="objectDefinition.spec.pruneObjectBehavior"
+                        label="Prune Object Behavior"
+                        labelHelp="Optional. Specifies how related objects on the managed cluster should be pruned when the policy is removed. The parameter values are None, DeleteIfCreated, and DeleteAll."
+                    >
+                        <Radio
+                            id="deleteIfCreated"
+                            label="Delete If Created"
+                            value="DeleteIfCreated"
+                            description="Attempts to remove objects known to be created by the policy when the policy is deleted."
```

Comment:
Attempts to delete objects that are created when the policy is deleted. 

Can you remind me of what objects are created after deletion?

Reply:
So we're distinguishing between the case where the ConfigurationPolicy *creates* a new object, vs when it just updates an existing one. We think there will be times when a user wants the policy to clean up after itself if it created the object, but not if the resource was already present on the cluster, because then some other process might be using it.

So the policy controller now specifically marks the objects as it creates them. Then if the policy is deleted and has `DeleteIfCreated` set, only the objects it knows it created will be deleted.